Install Fail on Windows

Installation attempt via:
`pip3 install git+https://github.com/src-d/lapjv`

This _appears_ to be a continuation of __cpuidex problems

```
Installing collected packages: lapjv
  Running setup.py install for lapjv ... error
  error: subprocess-exited-with-error

  × Running setup.py install for lapjv did not run successfully.
  │ exit code: 1
  ╰─> [22 lines of output]
      running install
      C:\Users\USER\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\setuptools\command\install.py:34: SetuptoolsDeprecationWarning: setup.py install is deprecated. Use build and pip and other standards-based tools.
        warnings.warn(
      running build
      running build_ext
      building 'lapjv' extension
      creating build
      creating build\temp.win-amd64-3.10
      creating build\temp.win-amd64-3.10\Release
      "C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\VC\Tools\MSVC\14.29.30133\bin\HostX86\x64\cl.exe" /c /nologo /O2 /W3 /GL /DNDEBUG /MD -IC:\Users\USER\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\numpy\core\include "-IC:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.1264.0_x64__qbz5n2kfra8p0\include" "-IC:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.1264.0_x64__qbz5n2kfra8p0\Include" "-IC:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\VC\Tools\MSVC\14.29.30133\include" "-IC:\Program Files (x86)\Windows Kits\NETFXSDK\4.7.2\include\um" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\shared" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\um" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\winrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.19041.0\cppwinrt" /EHsc /Tppython.cc /Fobuild\temp.win-amd64-3.10\Release\python.obj /openmp /std:c++latest /arch:AVX2
      python.cc
      C:\Users\USER\AppData\Local\Temp\pip-req-build-xf_k2cp2\cpuid.h(22): error C2664: 'void __cpuidex(int [],int,int)': cannot convert argument 1 from 'unsigned int [4]' to 'int []'
      C:\Users\USER\AppData\Local\Temp\pip-req-build-xf_k2cp2\cpuid.h(22): note: Types pointed to are unrelated; conversion requires reinterpret_cast, C-style cast or function-style cast
      C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\VC\Tools\MSVC\14.29.30133\include\intrin.h(161): note: see declaration of '__cpuidex'
      C:\Users\USER\AppData\Local\Temp\pip-req-build-xf_k2cp2\cpuid.h(32): error C2664: 'void __cpuidex(int [],int,int)': cannot convert argument 1 from 'unsigned int [4]' to 'int []'
      C:\Users\USER\AppData\Local\Temp\pip-req-build-xf_k2cp2\cpuid.h(32): note: Types pointed to are unrelated; conversion requires reinterpret_cast, C-style cast or function-style cast
      C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\VC\Tools\MSVC\14.29.30133\include\intrin.h(161): note: see declaration of '__cpuidex'
      C:\Users\USER\AppData\Local\Temp\pip-req-build-xf_k2cp2\cpuid.h(36): error C2664: 'void __cpuidex(int [],int,int)': cannot convert argument 1 from 'unsigned int [4]' to 'int []'
      C:\Users\USER\AppData\Local\Temp\pip-req-build-xf_k2cp2\cpuid.h(36): note: Types pointed to are unrelated; conversion requires reinterpret_cast, C-style cast or function-style cast
      C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\VC\Tools\MSVC\14.29.30133\include\intrin.h(161): note: see declaration of '__cpuidex'
      python.cc(127): warning C4244: 'initializing': conversion from 'npy_intp' to 'int', possible loss of data
      error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\BuildTools\\VC\\Tools\\MSVC\\14.29.30133\\bin\\HostX86\\x64\\cl.exe' failed with exit code 2
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: legacy-install-failure

× Encountered error while trying to install package.
╰─> lapjv

note: This is an issue with the package mentioned above, not pip.
hint: See above for output from the failure.
```

--- a/cpuid.h
+++ b/cpuid.h
@@ -1,10 +1,8 @@
 #pragma once
 
-#ifdef _WIN32
-  #ifdef _MSC_VER
-    #include<intrin.h>
-  #endif
-  #define CPUID(info, x)  __cpuidex(info, x, 0)
+#ifdef _MSC_VER
+  #include<intrin.h>
+  #define CPUID(info, x)  __cpuidex(reinterpret_cast<int *>(info), x, 0)
 #else
   #include <cpuid.h>
   #define CPUID(info, x)  __cpuid_count(x, 0, info[0], info[1], info[2], info[3])

